gamemodes/terrortown/gamemode/client/cl_keys.lua

```diff
@@ -86,7 +86,11 @@ function GM:PlayerBindPress(ply, bind, pressed)
 
 			return true
 		elseif TBHUD:PlayerIsFocused() then
-			return TBHUD:UseFocused()
+			if input.IsButtonDown(KEY_LALT) then
```

Comment:
not using the binding library isn't recommended, but I think it should be okay here... @saibotk

Reply:
For now that is okay, since the binding lib is not capable of key combinations